oauth2client/contrib/gce.py

```diff
@@ -51,13 +59,17 @@ class AppAssertionCredentials(AssertionCredentials):
     """
 
     @util.positional(2)
-    def __init__(self, scope, **kwargs):
+    def __init__(self, scope='', **kwargs):
         """Constructor for AppAssertionCredentials
 
         Args:
             scope: string or iterable of strings, scope(s) of the credentials
```

Comment:
How about just "any type, ignored" for a parameter description? Or "NoneType, a warning will be emitted for any non-None value but otherwise ignored"?